clients/tests/unit/render/util.test.js

```diff
@@ -1,13 +1,226 @@
-import {getFieldConfigBy} from "../../../render/util";
+import {
+	getFieldConfigBy,
+	removeFromBlocking,
+	removeFromPending,
+	removeFromUploadStarted,
+	setBlocking,
+	createMediaFromFile
+} from '../../../render/util'
+
+import * as data  from "./Mocks/mockUtils";
 import {formRenderTestProps} from "./CalderaFormsRender.test";
+
+
 describe(  'getFieldConfigBy', () => {
-	it ( 'finds fields that exits by fieldId', () => {
-		expect( getFieldConfigBy(formRenderTestProps.fieldsToControl, 'fieldId', 'fld_5899467' ).fieldIdAttr).toBe( 'fld_5899467_1' )
+
+	it('finds fields that exits by fieldId', () => {
+		expect(getFieldConfigBy(formRenderTestProps.fieldsToControl, 'fieldId', 'fld_5899467').fieldIdAttr).toBe('fld_5899467_1')
+	});
+
+	it('finds no fields when searching by invalid field id', () => {
+		expect(getFieldConfigBy(formRenderTestProps.fieldsToControl, 'fieldId', 'fld_1')).toBe(undefined)
+	});
+
+});
+
+describe.skip(  'Test files request', () => {
+
+	it( 'Calls to sub functions' , () => {
+
+		const createMediaFromFile = jest.fn();
+		const hashFile = jest.fn();
+
+		hashAndUpload(data.threeFiles, data.verify, data.field, data.fieldId, data.cf2, data.API_FOR_FILES_URL, data._wp_nonce, data.obj, createMediaFromFile, hashFile );
+
+		expect( createMediaFromFile ).toBeCalled();
+		expect( hashFile ).toBeCalled();
```

Comment:
@New0 I like that you're testing it calls the other functions, not the result of those functions. Just make sure those are tested elsewhere and that we get acceptance/integration tests later that cover everything together.